relnotes/continuetask.deprecation.md

```diff
@@ -0,0 +1,9 @@
+### Usage of `Task.continueAfterThrow` is not needed anymore
+
+`ocean.task.Task.continueAfterThrow` method has been deprecated and is now
+implemented as no-op. Instead a task which rethrows unhandled exception to the
+caller will be automatically resumed by scheduler for cleanup using
+`delayedResume` functionality.
```

Comment:
I don't quite get this sentence: "Instead a task which rethrows unhandled exception to the caller will be automatically resumed by scheduler for cleanup using `delayedResume` functionality"

Reply:
OK, I got what you want to say. I think it's just a missing comma issue :-).

"Instead, a task which rethrows ... will be automatically resumed ... "

Reply:
Suggestion:
"
`ocean.task.Task.continueAfterThrow` method has been deprecated and is now implemented as no-op. Calls to this method can simply be removed; the required cleanup behaviour is now handled automatically by the scheduler. (A task that rethrows an unhandled exception to the caller will now be automatically resumed by the scheduler for cleanup, using the `delayedResume` functionality.)
"